### DISCUSSION TITLE
Does this video still ok to watch (since it's a new language who evolve maybe fast)

### DISCUSSION THREAD

**Discussion by seb16120** [vlang/v#7697] [General]
Hi,
this video : https://www.youtube.com/watch?v=vcK1gLoa_bg

And do you know other video about this language ?

**Comment by larpon** [comment]
No it's old and already outdated IMO.
Currently there's official channels on twitch an youtube

**Reply by seb16120** [reply]
i found this more recent video but i don't thnink it's the official channel youy speak about :
https://www.youtube.com/watch?v=YpSsqRQLd4U

**Comment by larpon** [comment]
It's not the official, but people are free to post whatever.
